fr/index.org

```diff
@@ -1,99 +1,173 @@
 #+title:     Org mode for Emacs -- Votre vie en texte brut
-#+author:    Carsten Dominik
-#+email:     carsten at orgmode dot org
+#+author:    Julien Palard
+#+email:     julien at python dot org
```

Comment:
Just a little thing, we're actually removing author/email info from all pages (cleaner metadata, and on a lot of pages attribution isn't clear cut).

Reply:
Works for me.